OpenNefia.Content/UI/Layer/FieldLayer.cs

```diff
@@ -24,13 +25,14 @@ public partial class FieldLayer : UiLayerWithResult<UINone, UINone>, IFieldLayer
         [Dependency] private readonly IEntityManager _entityManager = default!;
         [Dependency] private readonly IMapRenderer _mapRenderer = default!;
         [Dependency] private readonly IMapManager _mapManager = default!;
-        [Dependency] private readonly IHudLayer _hud = default!;
         [Dependency] private readonly ICoords _coords = default!;
         [Dependency] private readonly IGameSessionManager _gameSession = default!;
         [Dependency] private readonly IGraphics _graphics = default!;
         [Dependency] private readonly IInputManager _inputManager = default!;
         [Dependency] private readonly IMusicManager _music = default!;
 
+        public override int? DefaultZOrder => 40000;
```

Comment:
Nitpick: `100000`, since I prefer rounder numbers.

Reply:
changed in 9eb449f